test/unit/utils/test_attrdict.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+
+from robot.utils.asserts import assert_equal, assert_not_equal, assert_raises
+from Selenium2Library.utils import AttrDict
+try:
+    from robot.utils import IRONPYTHON
+except ImportError:  # New in RF 2.9
+    import sys
+    IRONPYTHON = sys.platform == 'cli'
+    pass
```

Comment:
The above `try/except` should be removed because `IRONPYTHON` doesn't seem to be used for anything. Also, there's no need to have `pass` at the end of a non-empty block.